packages/iTwinUI-react/src/core/utils/components/VirtualScroll.tsx

```diff
@@ -242,6 +210,45 @@ export const useVirtualization = (props: VirtualScrollProps) => {
     };
   }, [visibleChildren.length]);
 
+  const onResize = React.useCallback(
```

Comment:
Minor, but could you move this variable back up to line 175 where it was previously? It makes it easier to review when the changes are side by side

Reply:
Ohh, I had tried doing that before, but I left it the current way because `onResize` now depends on `updateChildHeight`. So `onResize` now needs to be below `updateChildHeight`.

The lines after `onResize` 226-250 have to be in the order it is in since the next one is dependent on the previous one. However, I noticed that `getScrollableContainer` could be placed on line 175 to avoid an unnecessary change. Thus, I made that small [change](https://github.com/iTwin/iTwinUI-react/pull/811/commits/4e9b6f9eb11ae675960f0d00f5201a9d90eb7c1b)